invenio_rest/errors.py

```diff
@@ -112,16 +112,6 @@ class RESTValidationError(RESTException):
     """Error description."""
 
 
-class RESTCSRFError(RESTException):
```

Comment:
Was this needed to fix tests? If not, I would not remove this - even if it's unused across our repos. It can be used by other repositories that are not ours (e.g. partners). 

This change makes the PR need a major release, instead of a patch.

Reply:
Sure, good point, done!